front/app/containers/Admin/pagesAndMenu/components/SectionFormWrapper.tsx

```diff
@@ -8,14 +8,14 @@ import PageTitle from 'components/admin/PageTitle';
 import PageWrapper from 'components/admin/PageWrapper';
 import { SectionDescription } from 'components/admin/Section';
 
-interface Props {
+type Props = {
   breadcrumbs?: TBreadcrumbs;
   title?: string | JSX.Element;
   subtitle?: string | JSX.Element;
   children: JSX.Element | JSX.Element[];
   stickyMenuContents?: JSX.Element | JSX.Element[];
   rightSideCTA?: JSX.Element | JSX.Element[];
-}
+};
```

Comment:
I just made [this comment](https://github.com/CitizenLabDotCo/citizenlab/pull/2613#discussion_r962328986) on Iva's PR. Not sure why are converting Props to types but curious to hear the reasoning behind it.

Reply:
You very likely won't need to extend the local props for a component, so having it as a Type and not Interface signals that it's local and not likely to be extended elsewhere. No strong opinion about it though

Reply:
If you don't export your interface, it can't get extended in a module (only in the same file) so that already signals it' s local. :-)

Also, we use Interface for props as the default. If you search the codebase, you can find a 8:1 ratio vs type for props, so it seems a bit unnecessary to start converting it for no good reason. In fact, I only see a downside: it can confuse new people learning TS/the codebase who wouldn't understand why you need a type instead of an interface that they see in most places.

Reply:
Ok will change back